nerdlandbot/commands/minecraft.py

```diff
@@ -1,3 +1,4 @@
+from random import sample
```

Comment:
Is this necessary?

Reply:
Nope. Some stupid VScode setting most likely.